.travis.yml

```diff
@@ -1,10 +1,9 @@
 sudo: false
+
 language: node_js
-node_js:
-- "5.6"
 
-env:
-- TRAVIS=1
```

Comment:
Delete this bit, since I think this is true by default, per https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

> The following default environment variables are available to all builds.

> `CI=true`
> `TRAVIS=true`
> ...